kbase-extension/kbase_templates/page.html

```diff
@@ -54,8 +53,15 @@
       function gtag() { dataLayer.push(arguments); }
       gtag('js', new Date());
       gtag('config', '{{google_analytics_id}}', {
-        'cookie_domain': 'kbase.us'
-      })
+        'username': '{{userName}}',
+        'Page_location': document.location,
+        'page_path': document.location.pathname,
+        'page_title': document.location.pathname
+      });
+      gtag('set', {'user_id': '{{userName}}'});
+      if (document.location.hostname.includes('narrative')){
+        gtag('config', 'AW-753507180');
```

Comment:
So that only gets set in production? And shouldn't be set otherwise?

Reply:
AW tag is for google Ads so it won't be any good used for what internal dev humans do....